src/rotator_library/providers/anthropic_provider.py

```
@@ -0,0 +1,948 @@
+                "object": "chat.completion",
+                "created": first_chunk.created,
+                "model": first_chunk.model,
+                "choices": [
+                    {
+                        "index": 0,
+                        "message": final_message,
+                        "finish_reason": finish_reason,
+                    }
+                ],
+                "usage": usage_data,
+            }
```

Ruff (0.15.4)

[warning] Unnecessary `dict` kwargs

Remove unnecessary kwargs

(PIE804)